kitsune/sumo/tests/test_parser.py

```diff
@@ -23,9 +23,9 @@ def pq_img(p, text, selector='img', locale=settings.WIKI_DEFAULT_LANGUAGE):
     return doc(selector)
 
 
-def doc_rev_parser(content, title='Installing Firefox', parser_cls=WikiParser):
+def doc_rev_parser(content, title='Installing Firefox', parser_cls=WikiParser, **kwargs):
     p = parser_cls()
-    d = document(title=title)
+    d = document(title=title, **kwargs)
```

Comment:
I started to refactor this function, but it ended up having too much using it. Instead I tried to make as few changes to it as possible.